pkg/cluster/defaults_test.go

```diff
@@ -0,0 +1,116 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	testdatabase "github.com/Azure/ARO-RP/test/database"
+)
+
+func TestEnsureDefaults(t *testing.T) {
```

Comment:
Added this test to make sure database gets populated with the values. It bit overlaps with api tests but goal here is different

Reply:
Not sure how I feel about this test. You are basically testing that `api.SetDefaults` and `PatchWithLease` are working correctly. They should have tests on their own.

IMO `TestEnsureDefaults` should be testing:

1. What happens if there is an error returned from `PatchWithLease`.
2. What happens if there is no error returned from `PatchWithLease`.

Reply:
Ok, As I was on the fence with this test, will remove. we don't test this behavior anywhere else as this basically fails to install as database is down.